batcher/aligned/src/main.rs

```diff
@@ -325,6 +364,88 @@ async fn main() -> Result<(), errors::BatcherClientError> {
                     .map_err(|e| BatcherClientError::IoError(output_file.clone(), e))?;
             }
         }
+        DepositToBatcher(deposit_to_batcher_args) => {
+            if !deposit_to_batcher_args.amount.contains("ether") {
+                error!("Amount should be in the format XX.XXether");
+                return Ok(());
+            }
+
+            let amount = deposit_to_batcher_args.amount.replace("ether", "");
+
+            let eth_rpc_url = deposit_to_batcher_args.eth_rpc_url;
+
+            let eth_rpc_provider = Provider::<Http>::try_from(eth_rpc_url).map_err(|e| {
+                BatcherClientError::EthError(format!("Error while connecting to Ethereum: {}", e))
+            })?;
+
+            let keystore_path = &deposit_to_batcher_args.keystore_path;
+
+            let mut wallet = if let Some(keystore_path) = keystore_path {
+                let password = rpassword::prompt_password("Please enter your keystore password:")?;
+                Wallet::decrypt_keystore(keystore_path, password)?
+            } else {
+                warn!("Missing keystore used for payment.");
+                return Ok(());
+            };
+
+            match deposit_to_batcher_args.chain {
+                Chain::Devnet => wallet = wallet.with_chain_id(31337u64),
+                Chain::Holesky => wallet = wallet.with_chain_id(17000u64),
+            }
+
+            let client = SignerMiddleware::new(eth_rpc_provider.clone(), wallet.clone());
+
+            let balance = client
+                .get_balance(wallet.address(), None)
+                .await
+                .map_err(|e| {
+                    BatcherClientError::EthError(format!("Error while getting balance: {}", e))
+                })?;
+
+            if balance <= U256::from(0) {
```

Comment:
should we check it against the amount user want to send?
```suggestion
            if balance <= U256::from(amount) {
```

Reply:
Solved in [9c0b9b5](https://github.com/yetanotherco/aligned_layer/pull/514/commits/9c0b9b595ad463258932a545b4db07c9cf0b0810).